routes/artifacts.js

```diff
@@ -519,6 +523,7 @@ api.declare({
   method:     'get',
   route:      '/task/:taskId/runs/:runId/artifacts',
   name:       'listArtifacts',
+  stability:  base.API.stability.experimental,
```

Comment:
Basically, I'm not sure it's entirely stable yet...

For us to be able to iterate through all artifacts we likely have to support continuationToken...